pkg/system/hub/pgxhub/server.go

```diff
@@ -294,7 +294,7 @@ func (n *Server) TestHubNode(ctx context.Context, request *gen.TestHubNodeReques
 	_, err = client.GetMetadata(ctx, &traits.GetMetadataRequest{})
 	if err != nil {
 		logger.Debug("failed api request", zap.Error(err))
-		return nil, status.Error(codes.Unavailable, "failed api request")
+		return nil, status.Error(codes.Unavailable, fmt.Errorf("failed api request: %w", err).Error())
```

Comment:
I think you want `status.Errorf(codes.Unavailable, "failed api request: %v", err)`

Reply:
`%v` seemed to generic initially. The proper way to wrap errors is with `%w` but it only works in `fmt.Errorf` calls. I've amended.

Reply:
That's correct, but you aren't wrapping errors (as status.Error doesn't support that), you're formatting strings.